common/test/acceptance/tests/studio/test_studio_course_info.py

```diff
@@ -0,0 +1,318 @@
+"""
+Acceptance Tests for Course Information
+"""
+import uuid
+
+from bok_choy.web_app_test import WebAppTest
+from common.test.acceptance.pages.studio.course_info import CourseUpdatesPage
+from flaky import flaky
+
+from ...pages.studio.auto_auth import AutoAuthPage
+from ...pages.studio.index import DashboardPage
+from ...pages.studio.overview import CourseOutlinePage
+
+
+def _create_course(self):
+    """
+    Helper method to create a course within the setup method
+    """
+    self.auth_page.visit()
+    self.dashboard_page.visit()
+    self.dashboard_page.wait_for_page()
+    self.assertFalse(self.dashboard_page.has_course(
+        org=self.course_org,
+        number=self.course_number,
+        run=self.course_run
+    ))
+    self.assertTrue(self.dashboard_page.new_course_button.present)
+    self.dashboard_page.click_new_course_button()
+    self.assertTrue(self.dashboard_page.is_new_course_form_visible())
+    self.dashboard_page.fill_new_course_form(
+        self.course_name,
+        self.course_org,
+        self.course_number,
+        self.course_run
+    )
+    self.assertTrue(self.dashboard_page.is_new_course_form_valid())
+    self.dashboard_page.submit_new_course_form()
+
+    # Successful creation of course takes user to course outline page
+    course_outline_page = CourseOutlinePage(
+        self.browser,
+        self.course_org,
+        self.course_number,
+        self.course_run
+    )
+    course_outline_page.visit()
+    course_outline_page.wait_for_page()
+
+    # Go back to dashboard and verify newly created course exists there
+    self.dashboard_page.visit()
+    self.assertTrue(self.dashboard_page.has_course(
+        org=self.course_org, number=self.course_number, run=self.course_run
+    ))
+
+
+class UsersCanAddUpdatesTest(WebAppTest):
+    """
+      Scenario: Users can add updates
```

Comment:
Instead of having a class for each test point, I would have a single test method for each test point (so only a single class). The BDD-style description should go with the test point itself.